src/tests/ftest/launch.py

```diff
@@ -151,13 +152,22 @@ def set_test_environment(args):
         net_list = [dev for dev in os.listdir(net_path) if dev != "lo"]
         for device in sorted(net_list):
             # Get the interface state - only include active (up) interfaces
-            with open(os.path.join(net_path, device, "operstate"), "r") as fh:
-                state = fh.read().strip()
+            with open(os.path.join(net_path, device, "operstate"), "r") as \
+                 fileh:
+                state = fileh.read().strip()
```

Comment:
Small nit: renaming fh -> fileh seems like a disimprovement. It doesn't really improve comprehensibility and increases the line length past 80 chars so you have to add the line break.

Reply:
But pylint complains about `fh` as a variable name.  Too short IIRC.